src/pages/project.js

```diff
@@ -45,16 +60,55 @@ export const PageProject = ({ router }) => (
             <Breadcrumbs>
               <ProjectBreadcrumb projectSlug={project.name} />
             </Breadcrumbs>
-            <ProjectDetailsWrapper>
-              <div className="project-details-sidebar">
-                <ProjectDetailsSidebar project={project} />
-              </div>
-              <div className="environments-wrapper">
-                <h3>Environments</h3>
-                {!environments.length && <p>No Environments</p>}
-                <Environments environments={environments} project={project} />
-              </div>
-            </ProjectDetailsWrapper>
+            <ProjectWrapper>
+              <LeftNavTabs activeTab="overview" project={project} />
+              <ProjectDetailsWrapper>
+                <div className="project-details-header">
+                  <ProjectDetailsSidebar project={project} />
+                </div>
+                <div className="environments-wrapper">
+                  {!environments.length && <p>No Environments</p>}
+                  {environments.length &&
+                    environments.length > 0 &&
+                    getProductionEnvironments(environments) && (
+                      <div className="environments-production">
+                        <div className="environments-header">
+                          <div className="title">
+                            <h3>
```

Comment:
if you put a diff style on this h3 and the dev env one that reduces the lower margin from 36px to something else, that would bring the env cards up a bit higher